docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Migrating ASP.NET MVC Applications to Windows Containers
+description: Learn how to take an existing ASP.NET MVC application and run it in a Windows Docker Container
+keywords: .NET, .NET Server, .NET Desktop
+author: BillWagner
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-framework-4.6
+ms.technology: dotnet-mvc
+ms.devlang: dotnet
+ms.assetid: c9f1d52c-b4bd-4b5d-b7f9-8f9ceaf778c4
+---
+
+# Migrating ASP.NET MVC Applications to Windows Containers
+
+Running an existing .NET Framework-based application in a Windows container
+requires creating the Docker image that contains your application, and
+starting one or more containers to run that image. This topic explains
+the tasks you must perform to take an existing ASP.NET MVC application
+and deploy it in a Windows container.
+
+You'll start with an existing ASP.NET MVC application. You'll see how to
+build the assets to publish using Visual Studio. You'll use Docker
+to build the image that contains your application, and
+runs that application when it is started. You'll see how to connect to your
+application running in a Windows container.
+
+If you are unfamiliar with Docker, you can learn more about the Docker
+architecture by reading the 
+[Docker Overview](https://docs.docker.com/engine/understanding-docker/)
+on the Docker site. 
+
+The application you'll run in a container is a simple website that
+answers questions randomly. This application
+is a basic MVC application with no authentication support, or database
+storage. That lets you focus on moving the web tier to a container. Later
+topics show how to move and manage persistent storage in containerized
+applications.
+
+Moving your application involves these steps:
+
+1. [Creating a publish task to build the assets for an image.](#publish-script)
+2. [Building a Docker image that will run your application.](#build-the-image)
+3. [Starting a Docker container that runs your image.](#start-a-container)
+4. [Verifying the application using your browser.](#verify-in-the-browser)
+
+The finished application is located in the
+[dotnet/core-docs repository on GitHub](https://github.com/dotnet/core-docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator).
+
+## Prerequisites
+
+Your development machine must be running Windows 10 Anniversary Update, or newe,
+or, Windows Server 2016, or newer.
+
+Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/).
+You need to install version 1.12 Beta 26, or newer for Windows container support.
+After installing and starting Docker, you'll need to right-click on the
+tray icon and select **Switch to Windows containers...** in order to run
+Docker images based on Windows. This command takes a few seconds to
+execute:
+
+![Windows Container][windows-container]
+
+
+## Publish script
+
+You can use the Visual Studio Publish command to create a publish profile
```

Comment:
It might be useful to the reader to understand the why before the how. Also, I noticed in other articles we were making 'click' steps bold instead of putting quotes around them. I'm also suggesting bullet points to make it easier for the user to find their place when following along.

-----

You'll need to prepare an image that has your application before being able to deploy it into a Windows Container. Fortunately, you can use the Visual Studio Publish command to create a publish profile for your application. The publish command will put all the assets in one directory tree that you will copy to your target image later in this tutorial. 

 ** @Publish Steps**
1. Right-click on your solution in Visual Studio, and select **Publish**. 
2. Click the **Custom** profile button, and then select **File System** as the method. 
3. Choose a new directory in your solution folder. It is wroth noting that the downloaded sample uses "containerImage" as the folder name.